screens/signup.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Devstore | devlopers mall</title>
+    <link rel="stylesheet" href="../styles/gobal.css" />
+    <link rel="stylesheet" href="../styles/home.css" />
+    <link rel="stylesheet" href="../styles/auth.css" />
+  </head>
+  <body>
+    <main class="main-wrapper">
```

Comment:
this is the header section na. you can use header tag instead of main.

Reply:
Its not header @ashishkatiyar2404 its a main wrapper inside there is header which i am going to change after your suggestion to header thanks.